SunEngine.Admin/Controllers/RolesPermissionsAdminController.cs

```diff
@@ -37,11 +37,7 @@ public async Task<IActionResult> UploadJson(string json)
             }
             catch (Exception e)
             {
-                return BadRequest(new ErrorView
-                {
-                    ErrorName = e.Message,
-                    ErrorText = e.StackTrace
-                });
+                return BadRequest(new ErrorView(e.Message, e.StackTrace));
```

Comment:
I wasn't changed this code/message, but probably here we should have something else because returning stack trace to the user is a bad practice.